### ISSUE TITLE
adjust time out settings

### ISSUE THREAD

**Issue by janetbutts** [eHealthAfrica/direct-delivery-dashboard#331]
should be 15 minutes. looks like it is currently less than 2 minutes

**Comment by tlvince** [comment]
@janetbutts do you mean the time before you have to re-login if left idle? On develop, this is currently set at 10 minutes.

**Comment by janetbutts** [comment]
@tlvince , yes. when @yusufscratch and I were testing it, it would log out after a couple of minutes (def less than 5). we were testing planning, and it made us restart whatever we were testing several times. that would be super annoying in real life.

**Comment by tlvince** [comment]
I can't reproduce this; after logging in, I left https://dev-dd.ehealth.org.ng/ idle (for approximately 8 minutes) and was still able to move between pages without needing to login. CouchDB is set to expire the session after 10 minutes. I will test this is working. In the meantime, could you see if you can reproduce?

**Comment by tlvince** [comment]
After idling for ~12 mins, I was correctly redirected to the login page.

**Comment by janetbutts** [comment]
@yusufscratch , can you try and reproduce this from kano? i know we had different results when i shadowed you on the call.

**Comment by tlvince** [comment]
@yusufscratch please confirm on https://dev-dd.ehealth.org.ng/. If you can't reproduce the issue, feel free to close it. Otherwise, please reassign to me for further investigation. Thanks.

**Comment by janetbutts** [comment]
@yusufscratch, please test
